### PR TITLE
DOC: re-strucuture the doc folder

### PR DESCRIPTION
**Pull request by unkcpz**

### CODE REVIEW

**Comment by unkcpz** [comment]
Looks better now https://aiidalab-widgets-base--514.org.readthedocs.build/en/514/

File: docs/source/conf.py [hunk 1]
```diff
@@ -46,7 +41,7 @@
 
 # General information about the project.
 project = "aiidalab-widgets-base"
-copyright_first_year = "2020"
+copyright_first_year = "2023"
```
Comment (yakutovicha) [inline docs/source/conf.py:44]: Shouldn't it be the year the documentation was created?

**Review by yakutovicha — COMMENTED** [review]
Very minor question, otherwise good 👍

File: docs/source/conf.py (hunk 1 above)
Comment (unkcpz) [reply docs/source/conf.py:44]: You are correct, it didn't notice "first_year" :laughing:

Comment (unkcpz) [inline docs/source/conf.py:44]:
```suggestion
copyright_first_year = "2020"
```

**Review by yakutovicha — APPROVED** [review]
I actually need those changes very much, because I run into the same issues that you fixed here. So let's merge it 😎 

Thanks, @unkcpz!

**Pull request merged**